README.md

```diff
@@ -73,7 +74,6 @@ Example commands:
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you make your readme summary more detailed to make your readme descriptive and professional 👍